### ISSUE TITLE
[BUG] Alvar markers wrong positioning

### ISSUE THREAD

**Issue by ladrians** [luxonis/depthai-ros#326]
I am using the noetic branch from this [alvar marker](https://github.com/haraisao/ar_track_alvar/tree/noetic-devel) repository.

I have the OAK-D-W device and noticed that the marker detections are wrong located on the space, particularly when the markers are to the left or right side of the device. Created a minimal video to showcase it [here](https://github.com/luxonis/depthai-ros/assets/519869/720e8f40-fe8b-4b5a-a883-45ac7c4ef412). Notice the left marker shown as yellow, when rotating it goes away and it should not do that. With other cameras this problem is not detected.

The node does not need rectified images, I tested the left default topics (raw, rgb) and a rectified one created by the [rectify node](http://wiki.ros.org/image_proc) but all of them details similar wrong behavior.

Any thoughts on what to do to solve the case?

* Tried used the `*_i_calibration_file` parameters using the default file configuration but did not improved.
* Based on the [docs](https://docs.luxonis.com/projects/hardware/en/latest/pages/guides/calibration.html) it is not needed to do any camera recalibration, right?
* uploading the current [calibration](https://github.com/luxonis/depthai-ros/files/11642608/oakcalib01.txt)

thanks

**Comment by Serafadam** [comment]
Hi, I'll test that out, camera calibration should be okay if you are able to rectify the images properly or produce proper RGB pointcloud. Does this behavior also occur when using rviz to visualize the data?

**Comment by ladrians** [comment]
Yes, you can verify the problem happens easily with rviz. thanks
